### ISSUE TITLE
Cardiac tamponade

### ISSUE THREAD

**Issue by eswietlik** [obophenotype/human-phenotype-ontology#6547]
**Preferred term label:** Cardiac tamponade

**Synonyms**
Pericardial tamponade

**Definition (free text, please give PubMed ID)**
The accumulation of pericardial fluid with subsequent compression of all cardiac chambers which prevents cardiac filling. PMID: 12917306

**Parent term (use hpo.jax.org/app)**
Pericardial effusion HP:0001698

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0002-4095-8489

**Comment by pnrobinson** [comment]
added this, but as a child of https://hpo.jax.org/app/browse/term/HP:0011025
